dist/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>To Do List</title>
+  <script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+  <body>
+     <!--app body container-->
+    <div class="container row mx-auto px-3 py-5">
+      <!--app container-->
+      <div id="app-container" class="container-fluid row mx-auto bg-white border border-1 
+      shadow d-flex justify-content-center">
+        <!--header cols for title and refresh-->
+        <div class="row p-2 d-flex justify-content-between align-items-center">
+          <!--title col-->
+          <div class="col-10 pt-3">
+            <h1 id="app-title">Today's To Do</h1>
+          </div>
+          <!--refresh btn col-->
+            <i id="refresh-page-btn" class="bi bi-arrow-repeat btn btn-sm  col-1"></i> 
+        </div>
+        <hr>
```

Comment:
As an optional suggestion, it would be great if we could avoid using the HR tag in general.